gx_suite_definitions/target_exp_validation_harmonized.ipynb

```diff
@@ -0,0 +1,354 @@
+   "outputs": [],
+   "source": [
+    "# outcome_measure\n",
+    "validator.expect_column_values_to_be_of_type(\"outcome_measure\", \"str\")\n",
+    "validator.expect_column_values_to_be_in_set(\"outcome_measure\", [\n",
+    "        \"Behavioral\",\n",
+    "        \"Electrophysiology\",\n",
+    "        \"Biochemical\",\n",
+    "        \"Biomarker\",\n",
+    "        \"Cell Biology\",\n",
+    "        \"Other\",\n",
+    "        \"Histopathology\",\n",
+    "        \"Imaging\",\n",
+    "        \"Omics\",\n",
+    "        \"Immunochemistry\",\n",
+    "        \"Immunohistochemistry\",\n",
+    "    ]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# outcome_measure_details\n",
+    "validator.expect_column_values_to_be_of_type(\"outcome_measure_details\", \"str\")\n",
+    "validator.expect_column_value_lengths_to_be_between(\"outcome_measure_details\", 2, 2000)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# balanced_for_sex \n",
+    "validator.expect_column_values_to_be_of_type(\"balanced_for_sex\", \"str\")\n",
+    "validator.expect_column_values_to_be_in_set(\"balanced_for_sex\", [\"yes\", \"Yes\", \"no\", \"No\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# contributors\n",
+    "validator.expect_column_values_to_be_of_type(\"contributors\", \"str\")\n",
+    "validator.expect_column_value_lengths_to_be_between(\"contributors\", 2, 1000)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# team\n",
+    "validator.expect_column_values_to_be_of_type(\"team\", \"str\")\n",
+    "validator.expect_column_values_to_be_in_set(\"team\", [\n",
+    "    \"Duke\",\n",
+    "    \"Columbia-Rush\",\n",
+    "    \"MSSM - Zhang Lab\",\n",
+    "    \"Emory\",\n",
+    "    \"Mayo-UFL-ISB\",\n",
+    "    \"Mayo\",\n",
+    "    ]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# reference_doi\n",
+    "validator.expect_column_values_to_be_of_type(\"reference_doi\", \"str\")\n",
+    "validator.expect_column_value_lengths_to_be_between(\"reference_doi\", 2, 1000)\n",
+    "validator.expect_column_values_to_match_regex(\"reference_doi\", \"https://doi\\.org/[A-Za-z0-9/]+\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# date_report\n",
+    "validator.expect_column_values_to_be_of_type(\"date_report\", \"str\")\n",
+    "validator.expect_column_values_to_match_strftime_format(\"date_report\", \"%m/%d/%y\")\n",
+    "validator.expect_column_values_to_be_in_set(\"date_report\", [\"12/1/20\", \"12/1/23\"])"
```

Comment:
NIT: I don't think we need both `match_strftime_format` and `be_in_set` if the latter is checking specific dates already. I'm on the fence about which one to get rid of though.

Reply:
Oh right, I meant to include a comment asking which one you prefer. If those are the only dates we ever expect, then my thought was to go with `expect_column_values_to_be_in_set`

Reply:
We will be getting additional data in the future, with dates that are also in the future, so the `format` version will be easier to maintain.